modules/specification-level/src/main/scala/scala/cli/commands/SpecificationLevel.scala

```diff
@@ -65,6 +65,9 @@ object tags {
   val must: String           = SpecificationLevel.MUST.toString
   val should: String         = SpecificationLevel.SHOULD.toString
 
+  // other tags
+  val important: String = "important"
```

Comment:
Do we need a new tag? if so, it should be well described.

Reply:
And what is the difference between `Hidden` annotation and tag implementation?

 If we add `Hidden` annotation wherever there is no `implementation` tag, we will achieve the same effect.

Reply:
uh... shouldn't all options have a `SpecificationLevel` tag assigned, so that the [Runner Specification](https://scala-cli.virtuslab.org/docs/reference/scala-command/runner-specification) definition is complete?

Reply:
https://scala-cli.virtuslab.org/docs/reference/scala-command/runner-specification#implementantation-specific-options
certain options land in the `implementation specific` section there, no?

Reply:
as discussed offline, `tags.important` has been renamed to `tags.inShortHelp`